sonar-php-plugin/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 dependencies {
-  implementation(project(":php-frontend"))
+  api(project(":php-frontend"))
```

Comment:
Is it better instead to add `php-frontend` as a dependency in the custom plugins?